lib/tasks/accounts.rake

```diff
@@ -0,0 +1,74 @@
+namespace :accounts do
+  desc "merge two user accounts using environment variables SRC=<username-1> DST=<username-2>"
+  task merge: [:connection] do
+    if !ENV['SRC'] || !ENV['DST']
+      STDERR.puts "Usage: SRC=<username-1> DST=<username-2> rake accounts:merge"
+      exit 1
+    end
+
+    require 'exercism/user'
+
+    src = User.where(username: ENV['SRC']).first
+    if !src
+      STDERR.puts "Cannot find user record for %s" % ENV['SRC']
+      exit 1
+    end
+
+    dst = User.where(username: ENV['DST']).first
+    if !dst
+      STDERR.puts "Cannot find user record for %s" % ENV['DST']
+      exit 1
+    end
+
+    require 'exercism/user_exercise'
+    require 'exercism/named'
+    require 'exercism/problem'
+    require 'exercism/submission'
+    require 'exercism/acl'
+
+    src.exercises.each do |src_exercise|
+      # sometimes people have done the same exercise on both accounts
+      target = UserExercise.where(language: src_exercise.language, slug: src_exercise.slug, user_id: dst.id).first
+      if !!target
```

Comment:
Yeah, very much likely.  I don't see it used when it isn't guarding against returning a "truthy" return.  Also, I have had a conditional statement need to run millions of times in sequence, so no bottleneck of doing the `not not` twice.